util/src/types.rs

```diff
@@ -60,6 +62,7 @@ impl Default for LoggingConfig {
 			log_file_path: String::from("grin.log"),
 			log_file_append: true,
 			log_max_size: Some(1024 * 1024 * 16), // 16 megabytes default
+			log_max_files: Some(32),              // 32 log files to rotate over by default
```

Comment:
We use `32` in 2 places, would you consider introducing a constant?

Reply:
Done, thanks :-)